Description: No network access when building doc
 Removes the use of sphinx.ext.intersphinx
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-03-16

--- python-ceilometerclient-1.0.6.orig/doc/source/conf.py
+++ python-ceilometerclient-1.0.6/doc/source/conf.py
@@ -10,7 +10,7 @@ project = 'python-ceilometerclient'
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx']
+extensions = ['sphinx.ext.autodoc']
 
 # autodoc generation is a bit aggressive and a nuisance when doing heavy
 # text edit cycles.
